### PR TITLE
First commit to test code review

### PR DESCRIPTION
**Pull request by Gagatek5**
First commit to test code review.

### CODE REVIEW

File: FetaProject/FetaProject.Droid/Resources/Resource.designer.cs [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+//  <autogenerated>
+//      This code was generated by a tool.
+//      Mono Runtime Version: 4.0.30319.42000
+// 
+//      Changes to this file may cause incorrect behavior and will be lost if 
+//      the code is regenerated.
+//  </autogenerated>
+// ------------------------------------------------------------------------------
 
-[assembly: global::Android.Runtime.ResourceDesignerAttribute("FetaProject.Droid.Resource", IsApplication=true)]
```
Comment (PatrykSzwer) [inline FetaProject/FetaProject.Droid/Resources/Resource.designer.cs:12]: 1. Test comment:  That change shouldn't be done.

File: FetaProject/FetaProject.iOS/Info.plist [hunk 2]
```diff
@@ -46,5 +46,7 @@
 	<string>Resources/Images.xcassets/AppIcons.appiconset</string>
 	<key>XSLaunchImageAssets</key>
 	<string>Resources/Images.xcassets/LaunchImage.launchimage</string>
+	<key>CFBundleName</key>
```
Comment (PatrykSzwer) [inline FetaProject/FetaProject.iOS/Info.plist:49]: Second test comment: **Nice work!**

File: FetaProject/FetaProject.Droid/Resources/Resource.designer.cs (hunk 1 above)
Comment (PatrykSzwer) [inline FetaProject/FetaProject.Droid/Resources/Resource.designer.cs:12]: Test conversation: Hello!

**Review by PatrykSzwer — CHANGES_REQUESTED** [review]
See the comments below.

**Review by PatrykSzwer — APPROVED** [review]

File: FetaProject/FetaProject.Droid/Resources/Resource.designer.cs (hunk 1 above)
Comment (Gagatek5) [reply FetaProject/FetaProject.Droid/Resources/Resource.designer.cs:12]: OK

**Pull request merged**